### DISCUSSION TITLE
Difference between version from source and ready to use binary

### DISCUSSION THREAD

**Discussion by rzadki22** [OpenRadioss/OpenRadioss#2730] [Q&A]
Hello !

I wonder what is main difference between available versions of OpenRadioss from an user point of view. Openradioss might be built from source code either on Windows or Linux, from the other side one can use ready-to-use binary execs instead. Is it any speed difference or any other between two approaches? I expect that compiled version might be a bit faster. Do you have a comparison?
Best regards,
FJ

**Comment by MarianBulla** [comment]
Hello @rzadki22,
and thanks for your interest on OpenRadioss :-) 

Basiucally there is no difference between OpenRadioss precompiled executables vs. self compiled executables.

If you just want to use OpenRadioss for modelling of your physics, then go with the precompiled version.
I you want to change/modify/extend some material models and/or add other features, then just do it, and compile your own executables.

Hope it helps you ?

Thanks and best regards,
Marian
